lang/cs/Org.Apache.REEF.Bridge/Org.Apache.REEF.Bridge.nuspec

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<package >
```

Comment:
This will be addressed in [REEF-210](https://issues.apache.org/jira/browse/REEF-210)